VMAccess/vmaccess.py

```diff
@@ -65,9 +65,12 @@ def enable():
     hutil = Util.HandlerUtility(waagent.Log, waagent.Error, ExtensionShortName)
     hutil.do_parse_context('Enable')
     try:
+	public_settings = hutil._context._config['runtimeSettings'][0]['handlerSettings'].get('publicSettings')
         protect_settings = hutil._context._config['runtimeSettings'][0]['handlerSettings'].get('protectedSettings')
```

Comment:
it looks like an extra space snuck in here. I would be nice to clean that up to avoid unintentional consequences in later changes.